modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -816,6 +827,10 @@ GrainTracker::trackGrains()
       }
     }
 
+    // The reason for _t_step > 2 is to allow reasonable refinement of the mesh at grain boundaries.
+    if (_merge_grains_based_misorientaion && _t_step > 2)
```

Comment:
Similar comment as above. You'll want to change _t_step to just being an offset from the `_tracking_step` instead.

Reply:
Yes, the same modification as above